src/models/orderbook.rs

```diff
@@ -38,31 +38,27 @@ impl OrderBook {
     }
     #[allow(dead_code)]
     pub fn add_order(&mut self, order: Order) -> bool {
-        let mut current_orderbook = self.orders.write();
+        let mut current_orderbook = self.orders.write().unwrap();
```

Comment:
Maybe short comment why unwrapping here (and in the callsites below) is fine.

Reply:
In general, prefer `expect` over `unwrap` in these cases:
```suggestion
        let mut current_orderbook = self.orders.write().expect("rw lock can never be poisoned");
```